src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bars">
+      <form class="search-bar__form form-search-bar--big"
+        action="#"
+        data-qa="big"
+        method="post">
+        <input class="search-bar__input input-search-bar--big"
+        type="text"
+        name="bigSurchBar"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
+      </form>
```

Comment:
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) number 4, 5

Reply:
not fixed